variables.tf

```diff
@@ -210,4 +210,12 @@ variable "monitoring_service" {
 variable "service_account" {
   description = "The service account to default running nodes as if not overridden in `node_pools`. Defaults to the compute engine default service account"
   default     = ""
-}
\ No newline at end of file
+}
+
+variable "oauth_scopes" {
+  type = "list"
+  description = "The oauth scope the GKE compute nodes are allowed to have"
```

Comment:
Maybe stick to the same wording here: https://www.terraform.io/docs/providers/google/r/container_cluster.html#oauth_scopes

At least s/scope/scopes/